src/interpreter.coffee

```diff
@@ -0,0 +1,118 @@
+_ = window?._ ? self?._ ? global?._ ? require 'lodash'
+
+
+updateState = (aether, evaluator) ->
+  frame_stack = evaluator.frames
```

Comment:
Preference for `frameStack` for consistency with `lowerCamel` casing throughout rest of Aether.